src/odins_spear/endpoints/announcements.py

```
@@ -0,0 +1,240 @@
+            name (str): Desired name of the Announcement
+            description (str): Description assigned to the Announcement.
+            file_path (str): File path to the Announcement Audio File. (Raw String)
+            media_type (str): Audio file format
+
+        Returns:
+            None: This method does not return any specific value.
+        """
+
+        endpoint = "/users/announcements"
+
+        with open(file_path, "rb") as audio_file:  # Converts input audio file into b64
+            audio_bytes = audio_file.read()
+            b64_audio_bytes = base64.b64encode(audio_bytes)
+            content = b64_audio_bytes.decode("ascii")
+
+        data = {
+            "userId": user_id,
+            "name": name,
+            "mediaType": media_type,
+            "description": description,
+            "content": content,
+        }
+
+        return self._requester.post(endpoint, data=data)
+
+    def post_group_announcement(
+        self,
+        group_id: str,
+        service_provider_id: str,
+        name: str,
+        description: str,
+        file_path: str,
+        media_type: str = "WAV",
+    ):
+        """Adds an announcement to the given user.
+
+        Args:
+            group_id (str): Group ID of the target Group you would like to add the Announcement to.
+            service_provider_id (str): Service Provider ID of where the group resides
+            name (str): Desired name of the Announcement
+            description (str): Description assigned to the Announcement.
+            file_path (str): File path to the Announcement Audio File. (Raw String)
+            media_type (str): Audio file format
+
+        Returns:
+            None: This method does not return any specific value.
+        """
+
+        endpoint = "/groups/announcements"
+
+        with open(file_path, "rb") as audio_file:  # Converts input audio file into b64
```

Should we have a try catch here as the path the user gives us may not exist and we will need to return an exception